### PR TITLE
✨ Add amp-adplayer-pro

### PR DESCRIPTION
**Pull request by adp-developer**
An amp-adplayer-pro displays the video player by [AdPlayer.Pro](https://adplayer.pro/)

### CODE REVIEW

File: extensions/amp-adplayer-pro/validator-amp-adplayer-pro.protoascii [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+#
+
+tags: {  # amp-adplayer-pro
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-adplayer-pro"
+    version: "0.1"
+    version: "latest"
+    requires_usage: EXEMPTED
```
Comment (honeybadgerdontcare) [inline extensions/amp-adplayer-pro/validator-amp-adplayer-pro.protoascii:24]: please remove this line

File: extensions/amp-adplayer-pro/validator-amp-adplayer-pro.protoascii [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    deprecated_allow_duplicates: true
```
Comment (honeybadgerdontcare) [inline extensions/amp-adplayer-pro/validator-amp-adplayer-pro.protoascii:25]: please remove this line

**Comment by honeybadgerdontcare** [comment]
@alanorozco fyi

**Comment by jridgewell** [comment]
Sending this to @ampproject/wg-ui-and-a11y for review

**Comment by adp-developer** [comment]
@alanorozco Friendly ping!

**Comment by caroqliu** [comment]
Hi @adp-developer! Thanks for filing this issue, and sorry it's taken some time to get you a review on the component. All new components should follow the [process for significant changes](https://github.com/ampproject/amphtml/blob/master/contributing/contributing-code.md#process-for-significant-changes), including [filing an I2I](https://github.com/ampproject/amphtml/issues/new?assignees=&labels=INTENT+TO+IMPLEMENT&template=intent-to-implement--i2i-.md&title=I2I:%20%3Cyour%20change/update%3E) and [bringing the topic to design review](https://github.com/ampproject/amphtml/blob/master/contributing/design-reviews.md). Once this component has been approved at design review, we'll be sure to review this PR promptly.

**Comment by adp-developer** [comment]
Hello Caroline,

Thanks for getting back. Could you please let us know if we need to do
something else now?

**Comment by alanorozco** [comment]
@adp-developer Like @caroqliu mentioned:

> All new components should follow the [process for significant changes](https://github.com/ampproject/amphtml/blob/master/contributing/contributing-code.md#process-for-significant-changes), including [filing an I2I](https://github.com/ampproject/amphtml/issues/new?assignees=&labels=INTENT+TO+IMPLEMENT&template=intent-to-implement--i2i-.md&title=I2I:%20%3Cyour%20change/update%3E) and [bringing the topic to design review](https://github.com/ampproject/amphtml/blob/master/contributing/design-reviews.md)

**Review by alanorozco — CHANGES_REQUESTED** [review]
Reminder that this component is not approved, and its code cannot be reviewed until going to design review.

**Comment by alanorozco** [comment]
I strongly suggest that work on this is halted.

#33889 needs to go through a design review session first.

**Comment by Gregable** [comment]
Closing for now. It can be reopened once it has gone through design review.

**Pull request closed without merging**